### PR TITLE
Adopt chart for telegram bot

### PR DESCRIPTION
**Pull request by byymster**
## Description

Chart adoption for running telegram bot

## Type of change

Please make your selection.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a new dependency
- [ ] Code Maintainability / comments
- [ ] Code Refactoring / future-proofing

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added pytest unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by byymster** [comment]
@loomsen done

File: telegram_bot.py [hunk 1]
```diff
@@ -1564,7 +1600,7 @@ def main():
 
     # Get the dispatcher to register handlers
     dp = botconfig.updater.dispatcher
-
+    dp.run_async(botconfig.startallbotsoninit)
```
Comment (markhollingworth-worthit) [inline telegram_bot.py:1603]: Can you make this optional not everyone will wants starting automatically

Comment (byymster) [reply telegram_bot.py:1603]: sure

Comment (byymster) [reply telegram_bot.py]: done

**Comment by byymster** [comment]
All changes are in https://github.com/whittlem/pycryptobot/pull/589 too, maybe it's better to close this PR or it will be easier review !589 after merging this one

**Comment by whittlem** [comment]
@byymster, it's still not super clear to me what "chart" does or why we need it?

**Comment by whittlem** [comment]
I've reverted this PR. I just want to understand it a little better as there seems to be a lot more PR's to phase "chart" in. I am concerned about how many file changes there are and just don't want to introduce such a massive change which could introduce bugs and potentially lose people money. Maybe we should create another branch for this? Like "alpha" and test it properly first. "beta" will be merged into "main" after testing but it would be good to test this in "alpha" so we are not fully committed until we are comfortable with it.

**Comment by whittlem** [comment]
It doesn't look like I can re-open this PR. Can you open a new PR for this? 

I suggest pulling down beta, branch "alpha" off beta, then put your changes into "alpha". We can then use "alpha" to test your 3 PR's properly.

**Comment by whittlem** [comment]
I have created an "alpha" branch. Can you re-open this PR against alpha please? Thanks

**Comment by byymster** [comment]
Content of this PR also exists in #589 (it was impossible to do that changes without this code) so I propose to leave it as is and decide what to do with #589
Sorry that's not so transparent

**Pull request merged**